presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -99,6 +99,14 @@ String
 
     Example type definitions: ``varchar``, ``varchar(20)``
 
+    Data insertion in VARCHAR supports usage of unicode literals with the prefix
+    ``U&``, like ``U&`some unicode encoded literal data```. For example, you can
+    use ``U&```1F600``` for a grinning face emoji.
```

Comment:
Can you show an example with multiple characters? For example `hi😀`

Reply:
Done